src/core/perf_event/mod.rs

```diff
@@ -0,0 +1,35 @@
+#[cfg(any(
```

Comment:
I don't think you need the cfg block here, we seem to always include this particular module